tests/CI/check_db_initialized.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+dbMissing=true;
+allDBs=(JobDB FileCatalogDB)
```

Comment:
What is the rationale for this list?

Reply:
Well spotted, I need to dynamically find the DBs we install and make the list out of it, not hardcode this list that I used for tests

Reply:
https://github.com/DIRACGrid/DIRAC/blob/4d534a32168891f9ef6ce258dc41f668f5d2302a/tests/Jenkins/utilities.sh#L95-L99

Reply:
I think I will stick to a hard coded list, simply because we don't have DIRAC nor diracx installed in the container waiting for the DBs. So there is hardly any way to know which one you need to wait for. 
So I think it is safe to just put them pretty much all in it, and hope for the best. It can always be adjusted in case of need, as the error will be crystal clear